pypeit/par/pypeitpar.py

```diff
@@ -2155,7 +2155,8 @@ def __init__(self, dataext=None, dispaxis=None, xgap=None, ygap=None, ysize=None
         defaults['dispaxis'] = 0
         options['dispaxis'] = [ 0, 1, -1]
         dtypes['dispaxis'] = int
-        descr['dispaxis'] = 'Spectra are dispersed along this axis (0 for row, 1 for column)'
+        descr['dispaxis'] = 'Spectra are dispersed along this axis (0 for row, 1 for column,' \
```

Comment:
I think we'd rather have a new parameter that's specifically for flipping the spectral direction, like `dispflip` or something, because `0 == -0` is True.